setup.py

```diff
@@ -2,13 +2,40 @@
 
 setup(
     name='osmviz',
-    version='1.0',
+    version='1.0.1',
+    description=
+        'OSMViz is a small set of Python tools for retrieving '
+        'and using Mapnik tiles from a Slippy Map server '
+        '(you may know these as OpenStreetMap images).',
     long_description=open('README.md', 'r').read(),
-    packages= find_packages('src'),
-    package_dir = {'': 'src'},
-    zip_safe=False,
+    classifiers=[
+        #'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.5',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.1',
+        'Programming Language :: Python :: 3.2',
+        'Programming Language :: Python :: 3.3',
```

Comment:
Yeah, the code may well still work on those old versions, but I doubt they're used much, it at all, and it's not something I'd like to claim to or make the effort to support. Mainly because Travis CI doesn't support 2.5 or 3.1, and because 2.6 and 3.2 are getting harder to maintain as more and more other packages are dropping support (pip, coverage, etc.).